tests/backup/backup_kubevirt_test.go

```diff
@@ -2150,3 +2150,486 @@ var _ = Describe("{KubevirtVMRestoreWithAfterChangingVMConfig}", func() {
+			log.InfoD(fmt.Sprintf("Default restore of backups by replacing the existing resources"))
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			log.InfoD("Total backups to restore : %v", backupNames)
+			for i, bkpName := range backupNames {
+				restoreName = fmt.Sprintf("rreplace-%v-%s-%s", i, bkpName, RandomString(6))
+				//restoreNames = append(restoreNames, restoreName)
+				log.InfoD("Restoring from the backup - [%s]", bkpName)
+				err = CreateRestoreWithReplacePolicy(restoreName, bkpName, make(map[string]string), DestinationClusterName, BackupOrgID, ctx, make(map[string]string), 2)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Default restore of backups by replacing the existing resources [%s]", restoreName))
+				//replaceRestoreNames = append(replaceRestoreNames, restoreName)
+				//restoreNameBackupNameMap[restoreName] = bkpName
+			}
+		})
+
+		Step(fmt.Sprintf("Default restore of backups by replacing to a new namespace"), func() {
+			log.InfoD(fmt.Sprintf("Default restore of backups by replacing to a new namespace"))
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			for i, bkpName := range backupNames {
+				restoreName = fmt.Sprintf("rreplace-ns-%v-%s-%s", i, bkpName, RandomString(6))
+				//restoreNames = append(restoreNames, restoreName)
+				log.InfoD("Restoring from the backup - [%s]", bkpName)
+				actualBackupNamespaces, err := FetchNamespacesFromBackup(ctx, bkpName, BackupOrgID)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching namespaces from schedule backup %v - [%v]", actualBackupNamespaces, bkpName))
+				namespaceMapping := make(map[string]string)
+				for _, namespace := range actualBackupNamespaces {
+					namespaceMapping[namespace] = namespace + "-new"
+				}
+				log.InfoD("Backup namespace mapping : %v", namespaceMapping)
+				err = CreateRestoreWithReplacePolicyWithValidation(restoreName, bkpName, namespaceMapping, DestinationClusterName, BackupOrgID, ctx, make(map[string]string), 2, scheduledAppContexts)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Default restore of backups by replacing to a new namespace [%s]", restoreName))
+				//replaceRestoreNames = append(replaceRestoreNames, restoreName)
+				//restoreNameBackupNameMap[restoreName] = bkpName
+			}
+		})
+
+	})
+
+	JustAfterEach(func() {
+		defer EndPxBackupTorpedoTest(scheduledAppContexts)
+		defer func() {
+			log.InfoD("switching to default context")
+			err := SetClusterContext("")
+			log.FailOnError(err, "Failed to SetClusterContext to default cluster")
+		}()
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+		opts := make(map[string]bool)
+		opts[SkipClusterScopedObjects] = true
+		log.Info("Deleting backup schedules")
+		var wg sync.WaitGroup
+		var mutex sync.Mutex
+		errors := make([]string, 0)
```

Comment:
I think we are not handling these errors.

How about? *I have not tested this.

```go
errors := make([]string, 0)
for _, scheduleName := range scheduleNames {
	wg.Add(1)
	go func(scheduleName string) {
		defer GinkgoRecover()
		defer wg.Done()
		err = DeleteSchedule(scheduleName, SourceClusterName, BackupOrgID, ctx)
		dash.VerifySafely(err, nil, fmt.Sprintf("Deleting backup schedules [%s]", scheduleName))
		if err != nil {
			mutex.Lock()
			errors = append(errors, fmt.Sprintf("Failed while deleting schedules [%s]. Error - [%s]", scheduleName, err.Error()))
			mutex.Unlock()
		}
	}(scheduleName)
}
wg.Wait()
if len(erros) > 0 {
	err = fmt.Errorf("The combined list of errors while deleting backup schedules. Errs: [%v]", strings.Join(errors, ","))
	dash.VerifySafely(err, nil, "Verifying deletion of backup schedules")
}
```

Reply:
done